packer/images.json.pkr.hcl

```diff
@@ -151,6 +151,9 @@ source "virtualbox-iso" "cos" {
   ssh_timeout            = "5m"
   ssh_username           = "${var.root_username}"
   vm_name                = "cOS"
+  vboxmanage = [
+    ["modifyvm", "{{.Name}}", "--recording", "on", "--recordingscreens", "0","--recordingfile", "../capture.webm"],
```

Comment:
mmm just a silly question, is this going to record nevertheless when we ran the packer template? there is no way to optionally turn it off/on?

Reply:
no way of turning off for now. Do we want to skip it?

Reply:
I mean, we could just not do it in the packer or hide it under a debug env var I guess, packer is very consistent and usually never fails, and its easy to reproduce locally

Reply:
:+1: fine by me, was just curious.. I just wondered about webm support. Is something backed inside Vbox I guess, so should be fine!

Reply:
there, updated with a switch so its off by default

Reply:
Awesome! 👍